news/4580.bugfix

```diff
@@ -0,0 +1 @@
+Added an anchor tag, targeting a blank page!
```

Comment:
Give yourself credit.

```suggestion
Added an anchor tag, targeting a blank page. @falgunmpatel

```

Reply:
`git pull` to pull down the merge of `master` into your branch before you make the change. Then you can make the change, either in the GitHub UI or your editor, then commit and push.